test/csv.test.ts

```diff
@@ -27,15 +27,15 @@ describe("CSV parsing", function () {
 
   it("parses multiple claims per line", async function () {
     const stream = Readable.from(`Account,UserOption,Airdrop
-0x1234,42,1337`);
+0x4242424242424242424242424242424242424242,42,1337`);
     const expected: Claim[] = [
       {
-        account: "0x1234",
+        account: "0x4242424242424242424242424242424242424242",
```

Comment:
u used 42 because that's the meaning of life?

Reply:
Naturally. Did you notice this is PR #42? :smiley: 

Except for this, I think it's nice that if we see something that goes wrong somewhere with a lot of 42s, we can already start to check if some test data was introduced in the real data by accident.